acl_loader/main.py

```diff
@@ -471,7 +471,7 @@ def show_table(self, table_name):
                 if not val["ports"]:
                     data.append([key, val["type"], "", val["policy_desc"]])
                 else:
-                    ports = natsorted(val["ports"])
+                    ports = val["ports"].split(",")
```

Comment:
can you still sort after split?

Reply:
From the configuration Wiki page https://github.com/Azure/SONiC/wiki/Configuration, the "ports" field in "ACL_TABLE" should be list rather than string. List doesn't have split function and it will crash here.  I believe the old code works well if the port is list.

Reply:
@richard28530 Please check the code https://github.com/Azure/sonic-py-swsssdk/blob/7ce52d86101906e404af96fe76e04079f4afd863/src/swsssdk/configdb.py#L117-L120.
The var 'ports' is a list already. The original code is right.
Is it possible that you are testing with an old sonic-py-swsssdk?

Reply:
![image](https://user-images.githubusercontent.com/384871/48122438-80eef480-e2b2-11e8-9e70-08d924bff2ac.png)

I think the documentation is wrong.